.gitignore

```diff
@@ -8,4 +8,5 @@ __pycache__/
 .idea
 .vscode/
 *.egg-info/
-cos-tool-*
\ No newline at end of file
+cos-tool-*
+*.terraform*
```

Comment:
Not sure what we should standardise our .gitignores to for TF but I found this from one of our spec examples:

https://github.com/canonical/grafana-k8s-operator/pull/350/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947